homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
Please rename this file to index.html. This is the starting point for all static web sites.

Reply:
done